src/WindowsAzureMessaging/WindowsAzureMessaging/Models/MSInstallationTemplate.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+@interface MSInstallationTemplate : NSObject
+
+@property(nonatomic) NSString *body;
+@property(nonatomic) NSMutableSet<NSString *> *tags;
```

Comment:
We should strive not to expose mutable collections as properties as it breaks encapsulation.